code/modules/admin/view_variables/load_prefs.dm

```diff
@@ -0,0 +1,20 @@
+/mob/proc/vv_load_prefs()
+	if(!check_rights(R_ADMIN))
+		return
+
+	if(!client)
```

Comment:
Admins client

Reply:
> Might be worth making it something that only works on the admin's own mob, as it would be very un-fun for an admin to go around making everyone look like them. I know I've seen it used on skyrat a few times to create an army of granites in centcomm, or an army of selds, but I don't that use case really applies here.

If an Admin is going around doing that they should not be an admin in the first place. But in the context of fucking around in thunderdome or on the CC z level its fine imo

Reply:
> Admins client

Incorrect, it is the mob's client.